pkgs/tools/security/keybase/default.nix

```diff
@@ -1,8 +1,8 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, lib, buildGoPackage, fetchFromGitHub, darwin.apple_sdk.frameworks.AVFoundation ? null }:
```

Comment:
Oh but this is not even valid syntax. You should've noticed this locally already. Look at the file @xeji linked and replicate that in `all-packages.nix`, then this line here should just be `AVFoundation ? null`